sql-statements/sql-statement-create-user.md

```diff
@@ -29,6 +29,8 @@ AuthOption ::=
 StringName ::=
     stringLit
 |   Identifier
+
+LockOption ::= ( 'ACCOUNT' 'LOCK' | 'ACCOUNT' 'UNLOCK' )?
 ```
 
 ## 示例
```

Comment:
Can you provide an example using the `ACCOUNT LOCK` clause?

Reply:
Updated. PTAL.